core/templates/components/forms/schema-based-editors/schema-based-list-editor.directive.ts

```diff
@@ -81,6 +81,27 @@ angular.module('oppia').directive('schemaBasedListEditor', [
           return false;
         };
 
+        var oldTonewListMapping = {
+          newToOldListPosition: [],
+          deletedIndexes: []
+        };
+
+        var intializeOldTonewListMapping = function() {
```

Comment:
Here and elsewhere, "New" should be capitalized.

Reply:
done